middleware/recordChange.js

```diff
@@ -36,7 +36,7 @@ module.exports = function (req, res, next) {
 			}
 
 			// if transpose is defined for this field, let's tranpose their change so it can compare in the correct format on client side
-			if (form[fieldName].transpose) {
+			if (form[fieldName].transpose && typeof form[fieldName].transpose === 'function') {
```

Comment:
@allanchau, I think we need to extend this. At present, if you have `transpose.export`, you can't have a different transpose function specific to the `form` use?

I think we'll need to update this to have `transpose.export` and `transpose.form`... what do you think?

Reply:
yeah ok, sounds good.

Reply:
@allanchau, cool, thanks. Please update the documentation while you're at it outlining the different contexts... thanks.

Reply:
Maybe could write it like the following, so it's not a major change now (and we can deprecate it later?):

```
if (form[fieldName].transpose && typeof form[fieldName].transpose === 'function' || form[fieldName].transpose && form[fieldName].transpose.form && typeof form[fieldName].transpose.form === 'function')
```

Reply:
Well, if we're deprecating, not in one like that, because we'll need to do the deprecating stuff...

Reply:
i.e. https://www.npmjs.com/package/deprecate